_data/people/curi0sit7.yml

```diff
@@ -3,3 +3,5 @@ username: Curi0sit7
 name: Becca Flunker
 
 movie: Deadpool
+
+![Alt Text] (https://49.media.tumblr.com/fb6e1508fbfb355802a64d29e8ce42c7/tumblr_o4ru4g8s4q1qmoz1zo2_r2_500.gif)
```

Comment:
Right now, the site is only configured to show your [name](https://github.com/jasonrudolph/triangle-techgirlz-project/blob/ec74ef3f21c4e53582f259eec0cb42fa63e9aeab/index.html#L34) and your [movie recommendation](https://github.com/jasonrudolph/triangle-techgirlz-project/blob/ec74ef3f21c4e53582f259eec0cb42fa63e9aeab/index.html#L32). The site doesn't currently have a way to show an image for the movie, so I'm going to close this pull request.

However, I think it would be :cool: to show an image (e.g., the movie poster) for each movie. If you wanted to try enhancing the site to be able to show an image for each movie, you could open a pull request that updates [`index.html`](https://github.com/jasonrudolph/triangle-techgirlz-project/blob/ec74ef3f21c4e53582f259eec0cb42fa63e9aeab/index.html). If you have any questions, just let me know.